src/display/editor/tools.js

```diff
@@ -409,9 +416,10 @@ class AnnotationEditorUIManager {
    * Add a command to execute (cmd) and another one to undo it.
    * @param {function} cmd
    * @param {function} undo
+   * @param {boolean} mustExec
    */
-  addCommands(cmd, undo) {
-    this.#commandManager.add(cmd, undo);
+  addCommands(cmd, undo, mustExec = true) {
```

Comment:
Similar to another recent PR, can we please have the default value be `false` instead?
(Since having `undefined` => `true` feels quite surprising.)